recipes/repository.rb

```diff
@@ -10,13 +10,16 @@
   when 'apt'
     include_recipe 'apt::default'
 
+    apt_key = 'CBCB082A1BB943DB'
+    apt_key = 'F1656F24C74CD1D8' if node['platform'] == 'ubuntu' && node['platform_version'].split('.')[0].to_i >= 16
```

Comment:
seems like this could use a comment. is this change doc'd on the ubuntu website anywhere? is there a chance this will change again in future?